openradar/publishing.py

```diff
@@ -99,9 +99,11 @@ def publications(self, cascade=False):
         combinations = utils.get_product_combinations(
             datetimes=datetimes,
             prodcodes=self.prodcodes,
-            timeframes=self.timeframes,
+            timeframes=self.timeframes
         )
         for combination in combinations:
+            if self.nowcast:
```

Comment:
For every combination, also the nowcast product gets published. That leads to 9 times the same publication action on the round hours.